docs/community/contribution.md

```diff
@@ -68,6 +68,23 @@ If you find a bug in the source code, you can help us by submitting an issue to
 
 Or if you see errors in the documentation you can report them [here][github-docs-issue]
 
+Please do not label issue as a bug - we have to verify if the issue is not a feature, nor the thing that we haven't though of yet.
```

Comment:
Only delegated people (with access) can create issues. The trick is for issue to be accepted :)

Reply:
Good point